src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int hight;
+    private int bottomParallel;
+    private int topParallel;
+
+    public IsoscelesTrapezoid(String color, int hight, int topParallel, int bottomParallel) {
+        super(color);
+        this.hight = hight;
+        this.topParallel = topParallel;
+        this.bottomParallel = bottomParallel;
+    }
+
+    public int getHight() {
+        return hight;
+    }
+
+    public void setHight(int hight) {
+        this.hight = hight;
+    }
+
+    public double getTopParallel() {
+        return topParallel;
+    }
+
+    public void setTopParallel(int topParallel) {
+        this.topParallel = topParallel;
+    }
+
+    public int getBottomParallel() {
+        return bottomParallel;
+    }
+
+    public void setBottomParallel(int bottomParallel) {
+        this.bottomParallel = bottomParallel;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (bottomParallel + topParallel) / 2 * hight;
```

Comment:
(bottomParallel + topParallel) / 2: integer division in floating-point context.
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.